llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1652,6 +1694,29 @@ void VPReductionRecipe::print(raw_ostream &O, const Twine &Indent,
     O << " (with final reduction value stored in invariant address sank "
          "outside of loop)";
 }
+
+void VPReductionEVLRecipe::print(raw_ostream &O, const Twine &Indent,
+                                 VPSlotTracker &SlotTracker) const {
+  O << Indent << "REDUCE ";
```

Comment:
Better to have more explicit naming here. For example `VP-REDUCE`

Reply:
Following from other EVL recipes, vp.reduce is already there in the printing of reduction operation.
```
O << " vp.reduce." << Instruction::getOpcodeName(RdxDesc.getOpcode()) << " (";
```

Reply:
Since that is a different class, it makes more sense to actually have `VP-REDUCE` as a recipe name  and have `reduce` in rhs

Reply:
I prefer we should temporarily align with the existing EVL recipe. 
Later, we can gather all the developers involved with EVL vectorization to discuss how to standardize the print format for EVL recipes.

Reply:
Sounds good to me as first step, would be good to consistently identify EVL-based recipes in the printing. It would be great if you could share a follow-on PR for that.